app/main.py

```diff
@@ -9,16 +9,16 @@ def __init__(self, name: str, age: int) -> None:
 
 
 def create_person_list(people: list) -> list:
-    human_lst = [Person(human["name"], human["age"]) for human in people]
+    persons = [Person(person["name"], person["age"]) for person in people]
 
-    for human in people:
-        current_name = Person.people[human["name"]]
-        husband = human.get("husband")
-        wife = human.get("wife")
+    for name in people:
+        current_name = Person.people[name["name"]]
```

Comment:
no, `human un people` was good. I meant that it would be good to change `current_name` on `name` or `human_name`